github4s/src/main/scala/github4s/http/HttpClient.scala

```diff
@@ -27,9 +27,7 @@ import org.http4s.Request
 import org.http4s.circe.CirceEntityDecoder._
 import org.http4s.client.Client
 
-class HttpClient[F[_]: Sync](client: Client[F]) {
-
-  val urls: GithubAPIv3Config = GithubAPIv3Config()
+class HttpClient[F[_]: Sync](client: Client[F], val urls: GithubConfig) {
```

Comment:
```suggestion
class HttpClient[F[_]: Sync](client: Client[F], config: GithubConfig) {
```

Reply:
Oh... `val urls` is used from outside of `HttpClient`. Do you think it is better to rename its usages as well?

Reply:
yup :+1:

Reply:
fixed